insights_examples/rules/sshd_secure.py

```diff
@@ -57,20 +56,31 @@ def check_protocol(sshd_config, errors):
     return errors
 
 
-add_filter(Specs.sshd_config, ["AuthenticationMethods", "LogLevel", "PermitRootLogin", "Protocol"])
-
-
-@rule(InstalledRpms, SSHDConfig)
-def report(installed_rpms, sshd_config):
+@condition(SSHDConfig)
```

Comment:
@xiangce `@condition` is not explained in the document and this code doesn't match the code there.  I think it might be better to avoid any confusion and just keep everything in the `@rule` decorator here.  If you prefer to keep the `@condition` then I think you'll need to provide an explanation of how it works in the document, and update the code examples to match.

Reply:
Makes sense to me. I reverted the rule back to `no-@condition`. Thanks for reviewing @bfahr